docs/zkapps/tutorials/05-common-types-and-functions.mdx

```diff
@@ -39,6 +39,7 @@ The [example project](https://github.com/o1-labs/docs2/tree/main/examples/zkapps
 
 ## Prerequisites
 
+<<<<<<< HEAD
 This tutorial has been tested with [zkApp CLI](https://github.com/o1-labs/zkapp-cli) version `0.20.1` and [o1js](https://www.npmjs.com/package/o1js) version `1.1.0`.
```

Comment:
Let's use link to NPM package as we do for o1js: https://www.npmjs.com/package/zkapp-cli

Reply:
Or it can be handled as part of https://github.com/o1-labs/docs2/issues/926

Reply:
Let's update these all at once in another pr.